src/itinerary/Itinerary.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+  import { voyage } from "./../store/index.js";
+  import Map from "./Map.svelte";
+  import { Collection } from "sveltefire";
+
+  export const addExcursion = () => {
+    $voyage.collection("excursions").doc().set({ name: "test" });
```

Comment:
I know I put this here - but soon we're going to want to have the actual excursion name here.